src/index.test.js

```diff
@@ -114,6 +118,18 @@ test('console.error will not be called if disableLog is set', () => {
   getConfigForFile.throwError = null
 })
 
+test('console receives output of both eslintConfig and prettierOptions when sillyLogs is set', () => {
+  format.options.sillyLogs = true
+
+  // expect(format({text: ''})).toHaveBeenCalledWith(console.log, console.dir)
+  format({text: ''})
+  // TODO: fix this test, since it fails on the matcher toHaveBeenCalledTimes
+  expect(console.log).toHaveBeenCalledWith('😜 logs for eslintConfig and prettierOptions:')
+  expect(console.dir).toHaveBeenCalled()
```

Comment:
Could we be explicit with what it was called with as well as only having been called once?